manifests/kustomization.yaml

```diff
@@ -20,5 +20,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: cockroachdb/cockroach-operator
-  newName: quay.io/alinalion/cockroach-operator
-  newTag: v1.0.0-alpha.3
+  newName: us.gcr.io/chris-love-operator-playground/cockroach-operator
```

Comment:
What should this be?

Reply:
is the IMG field from the Makefile that is applied by bazel  at runtime. It should be the operator image for RED HAT.